fwdctl/tests/cli.rs

```diff
@@ -534,7 +533,76 @@ fn fwdctl_dump_with_hex() -> Result<()> {
         .stdout(predicate::str::contains("--start-key=c"))
         .stdout(predicate::str::contains("--start-key-hex=63"));
 
-    fwdctl_delete_db().map_err(|e| anyhow!(e))?;
+    fwdctl_delete_db()?;
+
+    Ok(())
```

Comment:
nit: trailing Ok(()):
```suggestion
    fwdctl_delete_db()
```